File: TubesWS/Object/Penerbit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TubesWS.Object
{
    public class Penerbit
    {
        private int id_penerbit;
        private string nama_penerbit; 
        private string lokasipercetakan;
        private string notelepon;

        public int Id_penerbit { get => id_penerbit; set => id_penerbit = value; }
        public string Nama_penerbit { get => nama_penerbit; set => nama_penerbit = value; }
        public string Lokasipercetakan { get => lokasipercetakan; set => lokasipercetakan = value; }
        public string Notelepon { get => notelepon; set => notelepon = value; }
    }
}

File: WSCI/application/views/penerbit.php

	<div class="container">
		<h2 align="center">Penerbit</h2>
			<table class="table table-hover">
				<thead>
					<th>ID</th>
					<th>Nama Penerbit</th>
					<th>Lokasi Percetakan</th>
					<th>No Telepon</th>
				</thead>
				<?php foreach($penerbit as $var){ ?>
				<tr>
					<td><?php echo $var['id_penerbit']; ?></td>
					<td><?php echo $var['nama_penerbit']; ?></td>
					<td><?php echo $var['lokasi_percetakan']; ?></td>
					<td><?php echo $var['notelepon']; ?></td>
				</tr>
				<?php } ?>
			</table>
	</div>

</body>
</html>